src/components/PropertyLookup.vue

```diff
@@ -34,23 +34,22 @@ export default Vue.extend( {
 			search: '',
 			searchResults: [],
 			topItemIndex: 1,
-			scrollpropertiesSearchResults: [],
 			searchOptions: {} as SearchOptions,
 		};
 	},
 	methods: {
 		handleScroll( event: number ): void {
-			if ( event <= this.topItemIndex ) {
+			if ( this.topItemIndex <= event ) {
+				this.topItemIndex += 12;
 				this.searchOptions.search = this.search;
```

Comment:
`searchOptions` should not be a field, but should be local variables. Both here and below in the watcher.
Currently, it remembers the search position across searches with different strings, but for every new search string it should start from the beginning again

Reply:
thanks for looking into that and checking it.
i made the changes as requested